test/data/xml.js

```diff
@@ -394,6 +418,29 @@ module.exports = {
 </h:html>`,
 
     updateEntity: `<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:entities="http://www.opendatakit.org/xforms">
+  <h:head>
+    <model entities:entities-version="2024.1.0">
+      <instance>
+        <data id="updateEntity" orx:version="1.0">
+          <name/>
+          <age/>
+          <hometown/>
+          <meta>
+            <entity dataset="people" id="" update="" baseVersion="" branchId="" trunkVersion="">
+              <label/>
+            </entity>
+          </meta>
+        </data>
+      </instance>
+      <bind nodeset="/data/name" type="string" entities:saveto="first_name"/>
+      <bind nodeset="/data/age" type="int" entities:saveto="age"/>
+    </model>
+  </h:head>
+</h:html>`,
+
+    // Copy of the above form with the original entities-version spec
+    updateEntity2023: `<?xml version="1.0"?>
```

Comment:
I kind of think it'd be nice not to duplicate the XML if we don't have to. What if we did something like:

```js
testData.forms.updateEntity2023 = testData.forms.updateEntity.replace('2024.1.0', '2023.1.0');
```

No strong feelings if you prefer to keep it the way it is.